vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/ApplicationSteps.java

```diff
@@ -83,12 +84,12 @@ public void startMobileApplicationWithCapabilities(List<NamedEntry> capabilities
     @Given("I start mobile application")
     public void startMobileApplication()
     {
-        HasCapabilities driverWithCapabilities = webDriverProvider.getUnwrapped(HasCapabilities.class);
-        LOGGER.atInfo()
-                .addArgument(
-                        () -> CapabilityHelpers.getCapability(driverWithCapabilities.getCapabilities(), "app",
-                                String.class))
-                .log("Started application located at {}");
+        Capabilities capabilities = webDriverProvider.getUnwrapped(HasCapabilities.class).getCapabilities();
+        String appCapability = CapabilityHelpers.getCapability(capabilities, "app", String.class);
+        if (appCapability != null)
+        {
+            LOGGER.info("Started application located at {}", appCapability);
```

Comment:
```suggestion
            LOGGER.atInfo().addArgument(appCapability).log(("Started application located at {}",);
```

Reply:
that's the matter of taste, IMHO the original version is more concise.